pytorch_lightning/core/lightning.py

```diff
@@ -57,15 +57,14 @@
 log = logging.getLogger(__name__)
 
 
-class LightningModule(
+class BaseLightningModule(
```

Comment:
Is this meant to be exposed publicly? If not, could we prefix it like this? 
```suggestion
class _BaseLightningModule(
```

assuming the other library would have a different top-level module that would be extended.

Reply:
Done !